### DISCUSSION TITLE
High memory causes app to exit，Lottie 4.3.4

### DISCUSSION THREAD

**Discussion by zhibinlee90** [airbnb/lottie-ios#2251] [General]
## Which Version of Lottie are you using?

Lottie 4.3.4

## Expected Behavior 
Animation plays normally

## Actual Behavior

![image](https://github.com/airbnb/lottie-ios/assets/15793177/3b01511b-8eaf-4549-bce3-7fb00aa636ff)

## Animation JSON
[mode10.json](https://github.com/airbnb/lottie-ios/files/13544194/mode10.json)

**Comment by calda** [comment]
This animation is very large (1.4MB) so there are a lot of CALayers that are created as a result. The only way to reduce the memory footprint of the animation would be to simplify it.

**Reply by zhibinlee90** [reply]
The same animation takes up less space when running on Android. Is this really unsolvable?
